src/Integration.Vsix.UnitTests/ErrorList/SonarErrorListDataSourceTests.cs

```diff
@@ -86,7 +89,7 @@ public void MefCtor_CheckExports()
                 // Both imports should be satisfied...
                 errorDataSourceImporter.Import.Should().NotBeNull();
                 issueLocationStoreImporter.Import.Should().NotBeNull();
-
+                clientIssueStore.Import.Should().NotBeNull();
```

Comment:
To check point (2) mentioned in the comment above, you could also add the following:

`
clientIssueStore.Import.Should().BeSameAs(errorDataSourceImporter.Import);
`